src/main/java/org/jenkinsci/plugins/testrail/JunitResults/Testcase.java

```diff
@@ -27,20 +27,24 @@
 public class Testcase {
     private String name;
     private Failure failure;
+    private Skipped skipped;
```

Comment:
The Skipped class is empty, I don't see how this would actually work.
I'm a CSharp dev who hasn't used Java regularly for almost a decade, would you be willing to educate me?